### PR TITLE
Branch Management handbook updates

### PR DESCRIPTION
**Pull request by Bubblemelon**
Resolves issue #573, and overall updates/fixes

### CODE REVIEW

File: release-team/role-handbooks/branch-manager/README.md [hunk 1]
```diff
@@ -44,6 +60,9 @@
    * ability to run `sendmail` from local Unix command line (BUG: should release-notify script be sending from GCB instead of the local machine, and can it?)
    * branchff: requires ability for your user to write /usr/local/ directory (BUG),
      sudo priv's, and membership in https://github.com/orgs/kubernetes/teams/kubernetes-release-managers
+* Join these mailing lists to keep updated:
+   * [kubernetes-sig-release](https://groups.google.com/forum/#!forum/kubernetes-release-team)
```
Comment (hoegaarden) [inline release-team/role-handbooks/branch-manager/README.md]:
```suggestion
   * [kubernetes-sig-release](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
```

Wrong link

File: release-team/role-handbooks/branch-manager/README.md [hunk 2]
```diff
@@ -28,13 +28,29 @@
...
 * [Patch release notes from exercising the new gcbmgr flow](https://docs.google.com/document/d/1x-GQDZpKk3WajtSnO0axDazE9Xs2mOSVgjziIuTWNO0/edit)
 
+# Overview
+The release branch manager is responsible for cutting a version of [Kubernetes](https://github.com/kubernetes/kubernetes/releases). Each release is a three month cycle where as branch manager:
+1. You will cut releases of v1.X as specified on the Timeline in `sig-release/releases/release-1.X/README.md`
+1. Participate in weekly release team meetings
+1. Commit about approx. 8hrs or less in a week of your time
+1. You will also get to select shadows and guide them in preparation to lead the next release cycle.
+
+For example, as branch manager of v1.15, the release cut dates as specified in the Timeline can be found at [`sig-release/releases/release-1.15/README.md`](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.15). The bulk of your time commitment is during release cut days.
```
Comment (hoegaarden) [inline release-team/role-handbooks/branch-manager/README.md]: One thing that is missing in this paragraph is the `branchff`. While a `branchff` individually does not take too long I think it is worth mentioning that this generally should be done on a daily basis (as soon as the branch was created). As generally shadows will probably not have push access to the GH issue, this remains the responsibility of the branch management lead.

I think that an important point regarding expected time commitment for that role.

File: release-team/role-handbooks/branch-manager/README.md [hunk 3]
```diff
@@ -28,13 +28,29 @@
...
+
+## Minimum Skills
+* Familiarity with basic Unix commands.
+* General knowledge of Google Cloud.
+* Able to debug Shell scripting.
+* Open to seeking help and can communicating clearly.
```
Comment (hoegaarden) [inline release-team/role-handbooks/branch-manager/README.md]: nit: Do we want/need to explicitly spell out that they would need access to a linux machine or a machine they can run linux containers or VMs on?

**Comment by Bubblemelon** [comment]
_Note to self/PR edits_: 
Include some kind of overview of the commands mentioned in this handbook. Similar to Tim's gist (first two columns from the left): https://gist.github.com/tpepper/29ceba2398858e5e6ff848d356a7d729

**Comment by Bubblemelon** [comment]
@idealhack Here's the handbook updates that I was talking about during yesterday's release cut zoom call.   :smiley:

File: release-team/role-handbooks/branch-manager/README.md [hunk 4]
```diff
@@ -164,11 +183,11 @@ branch](#branch-creation), the staging step will take about twice as long, the
...
+These require an additional layer of build and publish, which is currently still done by a Googler.  After building the official release, but before release notification, ping @sumi on Slack indicating for which build (for example: 1.12.0) Debs and RPMs are required.
 
 These build relatively quickly and should be available ahead of sending the release notification, especially for the official release when the worldwide community will attempt to get the new artifacts.  Since they build from the release tag, the first release step below is a pre-requisite for initiating the package builds.
 
-TODO: How do we automate this?  And in the meantime how do we ensure it is not bottlenecked on a one-on-one ping of a single Google employee (currently @sumi)?  And these should not just be made but also validated prior to the release-notify phase.
+TODO: How do we automate this?  And in the meantime how do we ensure it is not bottlenecked on a one-on-one ping of a single Google employee (currently @sumi)?  And these should not just be made but also validated prior to the release-notify phase. See issue: [/sig-release/#issues/10](https://github.com/kubernetes/sig-release/issues/10)
```
Comment (idealhack) [inline release-team/role-handbooks/branch-manager/README.md]: this should be updated to https://github.com/kubernetes/release/issues/631

File: release-team/role-handbooks/branch-manager/README.md (hunk 3 above)
Comment (idealhack) [reply release-team/role-handbooks/branch-manager/README.md]: this was listed here https://github.com/kubernetes/sig-release/pull/604/files#diff-52866a7ca0c8666623044eeb11e59015R55

**Pull request closed without merging**